Making Sense Project API/Controllers/CarController.cs

```diff
@@ -33,19 +41,40 @@ public IActionResult GetAll()
         [HttpDelete]
         public IActionResult Delete(int idCar)
         {
-            _carCRUD.DeleteById(idCar);
+            try
+            {
+                _carCRUD.DeleteById(idCar);
+            }
+            catch (System.Exception ex)
+            {
+                return BadRequest(ex.Message);
```

Comment:
No creo que sea bueno retornar detalles de un error interno.
```suggestion
                return BadRequest();
```